pancancer_classifier.py

```diff
@@ -500,12 +453,13 @@ def get_threshold_metrics(y_true, y_pred, tissue='all'):
                                       genes=alt_genes, loss_df=copy_loss_df,
                                       gain_df=copy_gain_df)
     # Append tissue id
-    y_alt = y_alt.assign(disease=clinical_df['acronym'])
```

Comment:
y_alt is a little difficult to follow -- it's a slice of mutation_df that has  the alternative genes classifier applied?